server/src/main/java/com/objectcomputing/checkins/services/reviews/ReviewPeriodServicesImpl.java

```diff
@@ -39,20 +43,27 @@ class ReviewPeriodServicesImpl implements ReviewPeriodServices {
     private final ReviewAssignmentRepository reviewAssignmentRepository;
     private final MemberProfileRepository memberProfileRepository;
     private final FeedbackRequestServices feedbackRequestServices;
+    private final FeedbackRequestRepository feedbackRequestRepository;
     private final ReviewStatusTransitionValidator reviewStatusTransitionValidator;
     private EmailSender emailSender;
     private final Environment environment;
     private final String webAddress;
 
+    private enum SelfReviewDate { LAUNCH, THREE_DAYS, ONE_DAY }
+    private final Map<String, Boolean> sent = new HashMap<>();
```

Comment:
I'm worried about this. We have multiple instances. What happens if this gets called once on each instance? Or even more likely, if the instance shuts down between calls? We may need to persist a given reminder has been sent for a given review period.

Reply:
Yes.  I thought this could be a problem.

Reply:
@ocielliottc yeah, I think that's a good call.